### DISCUSSION TITLE
Quest 2 support?

### DISCUSSION THREAD

**Discussion by Honosuseri** [CastagnaIT/plugin.video.netflix#1273] [Q&A] [answered]
I'm aware this VR headset has it's own Netflix app BUT I do like using Kodi and this excellent addon.

I've "sideloaded" kodi-19.3-Matrix-arm64-v8a onto the Quest 2 with SideQuest. Works great and I can watch my TV channels from my HDHomerun network TV tuner.

I can login to Netflix with this addon. I have to force the Widevine to L3 but only audio plays no image at all is shown. If anyone has a solution I'd love to hear it, thanks in advance.

**Comment by CastagnaIT** [comment, marked as answer]
possible solution for black video
When you playback a video on Android there is no video: https://github.com/CastagnaIT/plugin.video.netflix/wiki/FAQ-%28Audio%2C-Video%2C-Subtitle%2C-Other%29#When-you-playback-a-video-on-Android-there-is-no-video

if not works there are other incompatibilities with kodi and your system
could be media codec not detected, some kind of problem with widevine, or others...
but are not things that can be solved by this addon

**Reply by Honosuseri** [reply]
Thank you for replying and trying to help.

Yes I've tried those fixes before posting but no luck. I raised this as a discussion, not an issue with the add on itself. Just curious if other people had tried the add on with the Quest 2 VR HMD and if they had any success.

Once again thanks for your time trying to help me out. All the best
